projectile.el

```diff
@@ -3054,7 +3054,9 @@ test/impl/other files as below:
 ;; Racket
 (projectile-register-project-type 'racket '("info.rkt")
                                   :project-file "info.rkt"
-                                  :test "raco test .")
+                                  :test "raco test ."
+                                  :install "raco pkg install"
+                                  :package (concat "raco pkg create --source " (expand-file-name ".")))
```

Comment:
I don't think that `expand-file-name` is a good idea, as it's just going to be evaluated when Projectile loads. I guess a simple `.` is what you need.

Reply:
> I don't think that `expand-file-name` is a good idea, as it's just going to be evaluated when Projectile loads. I guess a simple `.` is what you need.

yea, that the catch, it has to be absolute path

Reply:
![image](https://user-images.githubusercontent.com/47485207/119449227-ea746880-bd21-11eb-8d9d-01c1ee88e34b.png)


![image](https://user-images.githubusercontent.com/47485207/119449271-f8c28480-bd21-11eb-860c-54e56fb6059c.png)